test/model/create.test.ts

```diff
@@ -30,6 +30,22 @@ test('creates a new entity with initial values', () => {
   expect(exactUser.id).toEqual('abc-123')
 })
 
+test('creates a new entity with an object property', () => {
```

Comment:
Let's add another test with the same setup but the `.create()` call where `.settings` wasn't specified. What would be the expected output in that case?

Reply:
Good Q. If we set `settings` as non-nullable, but don't specify a value to `.create()` method, I guess it should throw?

Reply:
Done, lemme know if that's not what you suggested - [02584dc](https://github.com/mswjs/data/pull/263/commits/02584dc0b4a02f108d97aa2bcabe0a54c2c4c68d)